### ISSUE TITLE
Declare the wiki as out of scope for the Editorial Board (PEP 732)

### ISSUE THREAD

**Issue by gvanrossum** [python/steering-council#232]
The Editorial Board has decided that the wiki is not in scope (at least not until we've made a lot more progress with other priorities) and would like to update PEP 732 to record this decision. It's a simple PR: https://github.com/python/peps/pull/3663

For background, see this Discourse thread: https://discuss.python.org/t/remove-the-link-to-the-old-python-wiki/45316

The process described in PEP 732 requires SC approval.

**Comment by gvanrossum** [comment]
(Note that the PR is in Draft mode to prevent it from being merged before SC approval. That's *all* we're waiting for.)

**Comment by gpshead** [comment]
This makes sense to the SC, go ahead.
